gcovr/templates/style.css

```diff
@@ -229,8 +305,68 @@ pre
     margin-bottom: 0;
 }
 
+.source-line
+{
+    height : 15px;
+    margin-top: 0;
+    margin-bottom: 0;
+}
 .lineno
 {
     background-color: #EFE383;
     border-right: 1px solid #BBB15F;
+    text-align: right;
+    display: block;
+    unicode-bidi: embed;
+    font-family: monospace;
+    white-space: pre;
+}
+
+/* new grid layout */
+.root-page
+{
+    display: grid;
+    font-family: sans-serif;
+}
+
+div.title
+{
+    grid-column: 1;
+
+    text-align: center;
+    padding-bottom: 10px;
+    font-size: 20pt;
+    font-weight: bold;
+}
+
+div.hr
+{
+    grid-column: 1;
+
+    background-color: navy;
+    height:3px;
+}
+
+.summary
+{
+    display: grid;
+    grid-column: 1;
+}
+
+.summary .legend
+{
+    grid-column: 1;
+}
+
+.summary .coverage
+{
+    grid-column: 2;
+}
+
+/* Footer format */
+.footer
+{
+  text-align: center;
+  padding-top: 3px;
+  font-family: sans-serif;
```

Comment:
Might as well set a global `* { font-family: sans-serif; }` instead of setting this for each element. (or should the selector for this be `:root` or `html`?)

Because the page title has been moved out of a table, it currently uses the browser default font (which is typically serif).

Reply:
done